src/synapse/azext_synapse/custom.py

```diff
@@ -88,8 +115,16 @@ def list_spark_session_statements(cmd, client, workspace_name, spark_pool_name,
     return client.list_statements(workspace_name, spark_pool_name, session_id)
 
 
-def create_spark_session_statement(cmd, client, workspace_name, spark_pool_name, session_id, code, kind):
-    livy_statement_request = LivyStatementRequestBody(code=code, kind=kind)
+def create_spark_session_statement(cmd, client, workspace_name, spark_pool_name, session_id, language, code=None,
+                                   code_file=None):

```

Comment:
I remember we can use one parameter for this.. Reference: [https://github.com/Azure/azure-cli/blob/8adf97355a605d147afc26f6c1b6f6afb6651c2f/src/azure-cli/azure/cli/command_modules/dla/_params.py#L62](https://github.com/Azure/azure-cli/blob/8adf97355a605d147afc26f6c1b6f6afb6651c2f/src/azure-cli/azure/cli/command_modules/dla/_params.py#L62)

Reply:
Thanks, changed.

---
In reply to: [417301532](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r417301532) [](ancestors = 417301532)